src/execution/execute.js

```diff
@@ -840,26 +832,18 @@ function completeValueCatchingError(
 function completeValueWithLocatedError(
   exeContext: ExecutionContext,
   returnType: GraphQLOutputType,
-  fieldNodes: $ReadOnlyArray<FieldNode>,
   info: GraphQLResolveInfo,
   path: ResponsePath,
   result: mixed,
 ): MaybePromise<mixed> {
   try {
-    const completed = completeValue(
-      exeContext,
-      returnType,
-      fieldNodes,
-      info,
-      path,
-      result,
-    );
+    const completed = completeValue(exeContext, returnType, info, path, result);
     if (isPromise(completed)) {
       return completed.then(undefined, error =>
         Promise.reject(
           locatedError(
             asErrorInstance(error),
-            fieldNodes,
+            info.fieldNodes,
```

Comment:
Similarly here - the `fieldNodes` value is an important argument to the `completeValue` functions - it should be passed directly instead of extracted from the info passed to resolvers